en/config/actuators.md

```diff
@@ -1,65 +1,170 @@
+This is expected to be enabled by default in PX4 v1.13 (currently disabled in `master`).
+
+To enable this feature, set the parameter [SYS_CTRL_ALLOC=1](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC) and make sure the correct frame type is set in [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+You may need to restart *QGroundControl*
 
-The easiest way to try this out in simulation is to use the following make target, which has control allocation pre-enabled:
+The easiest way to try this out in simulation is to use any of the following gazebo `make` targets, which have control allocation pre-enabled:
 ```
 make px4_sitl gazebo_iris_ctrlalloc
+make px4_sitl gazebo_typhoon_h480_ctrlalloc
 ```
 :::
 
-After selecting an [airframe](../config/airframe.md) you will generally need to configure the specific geometry, assign actuators to outputs, and test the actuator response.
-This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+Open the view in *QGroundControl* here: **"Q" (app menu) > Vehicle Setup > Actuators** (tab).
+
+## Overview
+
+The view has three sections:
+- [Geometry](#geometry): Set any configurable geometry-related parameters for the [selected airframe](../config/airframe.md).
+  This includes the number and relative position of motors, and the number, function, and properties of [control surfaces](#control-surfaces-geometry) (ailerons, rudders, etc.).
+- [Actuator Outputs](): Assign motors, control surfaces, and other actuators to specific flight controller outputs.
+- [Actuator Testing](): Test that motors and actuators are mapped to the correct outputs, and move as expected.
 
-The multicopter configuration screen looks like this.
+A quadcopter might have an setup screen similar to the one shown below.
+This defines a 4 motor copter with X-geometry, and maps each of the motors to DShot ESC.
+It also maps PWM400 AUX outputs for controlling a parachute and landing gear.
 
 ![Actuators MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_aux.png)
 
-Note that some settings are hidden unless you select the **Advanced** checkbox in the top right corner.
+Note that only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner to display all settings.
+
 
 ## Geometry
 
-The geometry section is used to configure any additional geometry-related settings for the selected [airframe](../config/airframe.md).
-The UI displays a customised view for the selected type; if there are no configurable options this may display a static image of the frame geometry, or nothing at all.
+The geometry section is used to set any configurable geometry-related parameters for the selected [airframe](../config/airframe.md).
+This includes the number and position of [motors](#motor-geometry), and the number, function, and properties of [control surfaces](#control-surfaces-geometry).
```

Comment:
We could omit the 'function' part of the Motors part, I am not sure what you mean by 'function'?